app/forms/sections/consignee/ConsigneeExportVatFormProvider.scala

```diff
@@ -27,6 +27,7 @@ class ConsigneeExportVatFormProvider @Inject() extends Mappings {
   def apply(): Form[String] =
     Form(
       "value" -> text("consigneeExportVat.error.required")
+        .transform[String](_.toUpperCase.replace(" ", ""), identity)
```

Comment:
Could those two transforms be combined and then a cheeky test to make sure " ", "-" and lower case characters get transformed correctly?